src/Core/Asset/GeometryData.hpp

```diff
@@ -232,49 +283,165 @@ class RA_CORE_API GeometryData : public AssetData
     /// Print stast info to the Debug output.
     void displayInfo() const;
 
+    /// Access to the multiIndexedGeometry;
+    inline Geometry::MultiIndexedGeometry& getMultiIndexedGeometry();
+
+    /// Access to the (const) multiIndexedGeometry;
+    inline const Geometry::MultiIndexedGeometry& getMultiIndexedGeometry() const;
+
     /// Access to the attrib manager
     inline Utils::AttribManager& getAttribManager();
 
     /// Access to the (const) attrib manager
     inline const Utils::AttribManager& getAttribManager() const;
 
+    /**
+     *
+     * @tparam T
+     * @param name
+     * @return Attrib<T> from m_multiIndexedGeometry.
+     * @note This function is only to avoid redundant code of function like getNormals().
+     */
+    template <typename T>
+    inline Utils::Attrib<T>& getAttrib( const Geometry::MeshAttrib& name );
+
+    /**
+     *
+     * @tparam V
+     * @param name
+     * @param firstOccurrence
+     * @return Initialize (if necessary) and get a VectorArray<V> contain in layer.
+     * @note This function is only to avoid redundant code of function like getEdges().
+     * If firstOccurrence == true, it allow to function to be more efficient but the name isn't use
```

Comment:
Why reference deprecated function ?

Reply:
That's a mistake, it was a previous version of this function that was deprecated. This one is not, I will remove this comment.